### PR TITLE
Feature/dashboard

### PR DESCRIPTION
**Pull request by lbmerchant93**
Creates the dashboard for student, will need to be modified to conditional render or be separated into own component based on if user if a student or alumni.

Installs fullcalendar and needed plugins

Tests for added functionality including:
- Calendar manipulation for next, today, prev, month view, week view, day view. 
- Information displayed for user name and timezone.
- User ability to select between 'behavioral' or 'technical' interviews

Adds type aliases back to tsconfig

Creates components for Prev/Next svgs, could be separated into different folder if team feels necessary.

### CODE REVIEW

File: app/dashboard/page.tsx [hunk 1]
```diff
@@ -1,7 +1,61 @@
...
+    id: "1",
+    email: "mockStudent@test.com",
+    userType: "student"
+}
+
 const Dashboard = () => {
+    const [interviewType, setInterviewType] = useState('');
+    const [isLoading, setIsLoading] = useState<boolean>(true);
+    //Caused error when not set to any
+    const calendarRef = useRef<any>();
```
Comment (Ant-Shell) [inline app/dashboard/page.tsx:21]: Will this any type need to be updated at some point?

File: components/CalendarHeaderToolbar.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
+import React, { FC, useState } from 'react';
+import moment from 'moment';
```
Comment (Ant-Shell) [inline components/CalendarHeaderToolbar.tsx:2]: We might need to revisit usage of the `moment` npm package at some point, per these resources:

- https://www.npmjs.com/package/moment
- https://momentjs.com/docs/#/-project-status/
- https://momentjs.com/docs/#/-project-status/recommendations/

**Review by Ant-Shell — COMMENTED** [review]
Looks great overall! Just had a few questions but we can discuss at our team meeting tomorrow.

File: app/dashboard/page.tsx (hunk 1 above)
Comment (lbmerchant93) [reply app/dashboard/page.tsx:21]: I'd love to change it to be more defined! If yall want to take a look at how to correct it, I'd appreciate the help

File: components/CalendarHeaderToolbar.tsx (hunk 2 above)
Comment (lbmerchant93) [reply components/CalendarHeaderToolbar.tsx:2]: I can look into using a different package before we merge this branch! Thank you for the resources, I didn't know they recommend this.

File: app/dashboard/page.tsx (hunk 1 above)
Comment (nkinsaul) [reply app/dashboard/page.tsx:21]: I can take a look at this later today.

**Review by Ant-Shell — APPROVED** [review]
Looks great! Approving and merging!

**Pull request merged**